lib/cretonne/src/context.rs

```diff
@@ -78,6 +80,12 @@ impl Context {
         self.domtree.compute(&self.func, &self.cfg);
     }
 
+    /// Perform simple GVN on the function.
+    pub fn simple_gvn(&mut self) -> CtonResult {
+        do_simple_gvn(&mut self.func, &mut self.cfg);
+        self.verify_if(None)
```

Comment:
This call doesn't do anything, with the `verify_if` implementation above.

I haven't thought much about tests without an ISA. It should always be possible to get at a `Flags` instance that we can give the context for those passes that don't need a full `TargetIsa`. See the definition of `IsaSpec` in the parser crate.